app/uk/gov/hmrc/gform/validation/ValidationService.scala

```diff
@@ -522,6 +522,13 @@ class ComponentsValidator(
     }
   }
 
+  private def postcodeValidation(fieldValue: FormComponent, fieldId: FormComponentId)(xs: Seq[String]): ValidatedType =
+    (xs.filterNot(_.isEmpty), fieldId) match {
```

Comment:
No need to pattern match on `fieldId` as it is not used.

Reply:
i have removed unused feildId within the pattern matching